modules/pricing/commerce_pricing.services.yml

```diff
@@ -0,0 +1,10 @@
+services:
+  commerce_pricing.chain_base_price_resolver:
+    class: Drupal\commerce_pricing\Resolver\ChainBasePriceResolver
+    tags:
+      - { name: service_collector, call: addResolver, tag: commerce_pricing.base_price_resolver }
+
+  commerce_pricing.default_base_price_resolver:
+    class: Drupal\commerce_pricing\Resolver\DefaultBasePriceResolver
+    tags:
+      - { name: commerce_pricing.base_price_resolver, priority: -100 }
```

Comment:
No ;) afaik Symfony weights in services are the opposite of the Drupal style weights - and I'm acutally using custom resolvers with a priority of "10"